### PR TITLE
sg: migrate jaeger and grafana to sg.config.yaml

### PR DESCRIPTION
**Pull request by mrnugget**
Didn't manage to get to `prometheus` and `postgres_exporter` yet, but prometheus should be similar to `grafana`

### CODE REVIEW

**Review by eseliger — APPROVED** [review]
🔍

File: sg.config.yaml [hunk 1]
```diff
@@ -500,6 +500,88 @@ commands:
...
+      IMAGE: sourcegraph/grafana:dev
+      CONTAINER: grafana
+      PORT: 3370
+      # docker containers must access things via docker host on non-linux platforms
+      DOCKER_USER: ""
+      ADD_HOST_FLAG: ""
+      DISABLE_SOURCEGRAPH_CONFIG: false
+
+  prometheus:
+    cmd: ./dev/prometheus.sh
```
Comment (efritz) [inline sg.config.yaml:580]: I thought this was a big no-no for sg?

**Review by efritz — APPROVED** [review]

File: sg.config.yaml (hunk 1 above)
Comment (efritz) [reply sg.config.yaml:580]: > Didn't manage to get to prometheus and postgres_exporter yet, but prometheus should be similar to grafana

LOL read the PR descriptions eric.

**Review by Strum355 — APPROVED** [review]

File: sg.config.yaml (hunk 1 above)
Comment (mrnugget) [reply sg.config.yaml:580]: Yeah, I'll do that next.

**Pull request merged**